src/client/core/aad/auth-provider.ts

```diff
@@ -63,45 +79,80 @@ export default class AuthProvider {
                     { ...authRequest, prompt: "none" }
                 );
                 code = await authCodeCallback(url, true);
-            } catch (e) {
+            } catch (silentAuthExcpetion) {
```

Comment:
Again, seems like this may be a good place to adding logging so we can troubleshoot auth issues in the future. Also minor typo on silentAuthException